### PR TITLE
fix missing ascend and try around experiment execution in director

### PR DESCRIPTION
**Pull request by MarcusZuber**
A test, if the walker is at the same position after the execution would be maybe something to check this also in the future...

### CODE REVIEW

**Comment by MarcusZuber** [comment]
What I wrote recently was not so ideal, but now everithing should work and tested...

File: concert/directors/scanning.py [hunk 1]
```diff
@@ -9,12 +9,12 @@ class XYScan(Director):
...
-    x_step = Quantity(q.mm, check=['standby', 'error'])
-    y_min = Quantity(q.mm, check=['standby', 'error'])
-    y_max = Quantity(q.mm, check=['standby', 'error'])
-    y_step = Quantity(q.mm, check=['standby', 'error'])
+    x_min = Quantity(q.mm)
+    x_max = Quantity(q.mm)
+    x_step = Quantity(q.mm)
+    y_min = Quantity(q.mm)
+    y_max = Quantity(q.mm)
+    y_step = Quantity(q.mm)
```
Comment (tfarago) [inline concert/directors/scanning.py:17]: Why are the checks gone?

Comment (MarcusZuber) [reply concert/directors/scanning.py:17]: I didn't check in detail, but I had the impression, that the check parameter requires a function and not a list of states. I will have a second look.

**Comment by tfarago** [comment]
:+1:

**Pull request merged**